### PR TITLE
Add 'lending_currentInterestRate' RPC method

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/29gp23n

## Description
I added `lending_currentInterestRate` RPC method. We can add it to RPC of node after we will add pallet-lending to `Picasso` and `Composable` runtimes.

### CODE REVIEW

File: frame/lending/src/lib.rs [hunk 1]
```diff
@@ -1773,4 +1775,27 @@ pub mod pallet {
...
+		let total_borrowed_from_market_excluding_interest =
+			Pallet::<T>::total_borrowed_from_market_excluding_interest(&market_id)?;
+		let total_available_to_be_borrowed =
+			Pallet::<T>::total_available_to_be_borrowed(&market_id)?;
+		let utilization_ratio = Pallet::<T>::calculate_utilization_ratio(
+			total_available_to_be_borrowed,
+			total_borrowed_from_market_excluding_interest,
+		)?;
+
+		Markets::<T>::try_mutate(market_id, |market_config| {
```
Comment (hussein-aitlahcen) [inline frame/lending/src/lib.rs:1791]: Why do we try mutate? If get borrow rate is mut then the storage will be mutated? Perhaps just try_get and mutate after, unless I missee something 🤔

Comment (andor0) [reply frame/lending/src/lib.rs]: Yes, you are right, we can use `try_get` here. Fixed.

File: frame/lending/src/lib.rs [hunk 2]
```diff
@@ -1773,4 +1775,24 @@ pub mod pallet {
 		pub(crate) accrued_increment: T::Balance,
 		pub(crate) new_borrow_index: FixedU128,
 	}
+
+	/// Retrieve the current interest rate for the given `market_id`.
+	pub fn current_interest_rate<T: Config>(market_id: MarketId) -> Result<Rate, DispatchError> {
```
Comment (vimukthi-git) [inline frame/lending/src/lib.rs:1780]: It would be best to write some unit test(s) for this function,

File: runtime/common/Cargo.toml [hunk 3]
```diff
@@ -36,6 +36,7 @@ collective = { package = "pallet-collective", git = "https://github.com/parityte
 treasury = { package = "pallet-treasury", git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.18" }
 
 composable-traits = { path = "../../frame/composable-traits", default-features = false }
+pallet-lending = { path = "../../frame/lending", default-features = false }
```
Comment (vimukthi-git) [inline runtime/common/Cargo.toml:39]: Seems unnecessary to import?

File: runtime/common/src/lib.rs [hunk 4]
```diff
@@ -26,6 +26,7 @@ pub use types::*;
 mod types {
 	use codec::{Decode, Encode, MaxEncodedLen};
 	use core::fmt::Debug;
+	pub use pallet_lending::MarketId;
```
Comment (vimukthi-git) [inline runtime/common/src/lib.rs:29]: Seems unnecessary to import?

File: frame/lending/rpc/src/lib.rs [hunk 5]
```diff
@@ -0,0 +1,66 @@
+use codec::Codec;
+use composable_support::rpc_helpers::SafeRpcWrapper;
+use composable_traits::defi::Rate;
+use core::{fmt::Display, str::FromStr};
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result as RpcResult};
+use jsonrpc_derive::rpc;
+use lending_runtime_api::LendingRuntimeApi;
```
Comment (vimukthi-git) [inline frame/lending/rpc/src/lib.rs:7]: It seems this runtime API has not been declared?

File: runtime/common/src/lib.rs (hunk 4 above)
Comment (andor0) [reply runtime/common/src/lib.rs]: Removed it.

**Pull request merged**